test/e2e/wizard.test.js

```diff
@@ -98,84 +98,104 @@ describe('vl-wizard', async () => {
     });
 
     it('als gebruiker kan ik via de actie knoppen alleen door de panelen van de wizard navigeren wanneer ik de checkbox aanzet', async () => {
-        const wizard = await vlWizardPage.getCheckboxWizard();
-        const pane1 = await wizard.getPane(1);
-        const pane2 = await wizard.getPane(2);
-        const pane3 = await wizard.getPane(3);
-
-        let activePane = await wizard.getActivePane();
-        await assert.eventually.isTrue(activePane.equals(pane1));
-        await wizard.next();
-        await activePane.equals(pane1);
-        let content = (await activePane.getContentSlotElements())[0];
-        let checkbox = await content.findElement(By.css('#vl-wizard-checkbox-pane-1-next'));
-
-        await checkbox.click();
-        await wizard.next();
-        activePane = await wizard.getActivePane();
-        await assert.eventually.isTrue(activePane.equals(pane2));
-        await wizard.previous();
-        await activePane.equals(pane2);
-        await wizard.next();
-        await assert.eventually.isTrue(activePane.equals(pane2));
-        content = (await activePane.getContentSlotElements())[0];
-        checkbox = await content.findElement(By.css('#vl-wizard-checkbox-pane-2-previous'));
-
-        await checkbox.click();
-        await wizard.previous();
-        activePane = await wizard.getActivePane();
-        await assert.eventually.isTrue(activePane.equals(pane1));
-        await wizard.next();
-        activePane = await wizard.getActivePane();
-        content = (await activePane.getContentSlotElements())[0];
-        checkbox = await content.findElement(By.css('#vl-wizard-checkbox-pane-2-next'));
-
-        await checkbox.click();
-        await wizard.next();
-        activePane = await wizard.getActivePane();
-        await assert.eventually.isTrue(activePane.equals(pane3));
+        const wizard1 = await vlWizardPage.getDisabledWizard();
```

Comment:
'wanneer ik de checkbox aanzet' is toch alleen 'toevallig' bij uw voorbeeld-wizard uit de testen? dus die naam lijkt me niet helemaal te kloppen in het grotere plaatje